README.md

```diff
@@ -115,7 +115,7 @@ Unpinning and removing "object-assign"
 
 ## Preloading pinned modules
 
-To avoid the waterfall effect where the browser has to load one file after another before it can get to the deepest nested import, importmap-rails uses [modulepreload links](https://developers.google.com/web/updates/2017/12/modulepreload) by default. If you don't want to preload a dependencies, because it'you want to load it on-demand for efficiency, pinned modules can prevent preloading by appending `preload: false` to the pin.
+To avoid the waterfall effect where the browser has to load one file after another before it can get to the deepest nested import, importmap-rails uses [modulepreload links](https://developers.google.com/web/updates/2017/12/modulepreload) by default. If you don't want to preload dependencies, e.g. because you want to load it on-demand for efficiency, pinned modules can prevent preloading by appending `preload: false` to the pin.
```

Comment:
I'd suggest we simplify this to:
```suggestion
To avoid the waterfall effect where the browser has to load one file after another before it can get to the deepest nested import, importmap-rails uses [modulepreload links](https://developers.google.com/web/updates/2017/12/modulepreload) by default. If you don't want to preload a dependency, because you want to load it on-demand for efficiency, append `preload: false` to the pin.
```
which removes the passive voice (which felt a bit awkward) in favor of the imperative tense and deletes the redundant part

Reply:
Yes, that's better. Thank you for the review and suggestion. I'll update shortly.